accelerator/config.h

```diff
@@ -37,11 +37,12 @@ extern "C" {
 #define IRI_HOST "localhost"
 #define IRI_PORT 14265
 #define MILESTONE_DEPTH 3
-#define FSIZE 11
 #define MWM 14
 #define SEED                                                                   \
   "AMRWQP9BUMJALJHBXUCHOD9HFFD9LGTGEAWMJWWXSDVOF9PI9YGJAPBQLQUOMNYEQCZPGCTHGV" \
   "NNAPGHA"
+#define MAM_FILE_DIR "/tmp"
```

Comment:
Don't depend on the prefix `/tmp`. Instead, use `mkstemp`: http://man7.org/linux/man-pages/man3/mkstemp.3.html

Reply:
`mkstemp` will create a file which result `fread` return non-NULL when reading this file's directory.
In this situation, `mam_api_load()` would return error. 
Or maybe, I could use a fixed file path for MAM file and encrypt it?

Reply:
We can assume the prefix of MAM file as `/tmp/`, but the real filename should be tracked inside TA.